src/backend/src/transformers/auth-user.transformer.ts

```diff
@@ -10,7 +11,8 @@ const authenticatedUserTransformer = (user: Prisma.UserGetPayload<typeof authUse
     email: user.email,
     emailId: user.emailId,
     role: user.role,
-    defaultTheme: user.userSettings?.defaultTheme
+    defaultTheme: user.userSettings?.defaultTheme,
+    teamAsLeadId: getTeamForLead(user.userId)
```

Comment:
all you need to do is change the query args to include the `teamAsLead` and then pull it out here